### ISSUE TITLE
Ability to keep scrolling until we have reached the end of the scroll container

### ISSUE THREAD

**Issue by cliffordfajardo** [davidcetinkaya/embla-carousel#171]
### Feature request is related to

- [X] The Vanilla JavaScript version
- [X] The React version
- [X] The Embla Carousel core (all versions)
- [ ] Documentation website
- [ ] Other

### Describe the solution you'd like
Ability to keep scrolling/dragging until I have reached the end of the scroll container.
Currently my item is getting truncated and I can't continue dragging to see the end of the item.

Explanation of the options I am using:

```
const options = {
    axis: 'x',
    align: 'start',   // align with value of `start` is required so that when we resize the browser, the slidelist doesnt have a negative transform value.
    dragFree: true, // when a user resizes their browser viewport on desktop to tablet-smaller devices this is setting is allows the user drag to see the overflowed item nicely.
  };
```

Perhaps we can have a new option to allow scrolling until the end?

#### My Demo Example:

Code Sandbox: 
- https://codesandbox.io/s/eager-snowflake-7zrpl?file=/src/App.tsx

** I have 3 slides. Each slide contains 3 items. ** 

https://user-images.githubusercontent.com/6743796/113490639-4e945200-9480-11eb-9545-425ef2d4fc02.mp4

![image](https://user-images.githubusercontent.com/6743796/113490919-0413d500-9482-11eb-957d-47abef553706.png)

Real world example of these slide this type of slide list behavior
1. LinkedIn's slide lists on the `linkedin.com/learning`

https://user-images.githubusercontent.com/6743796/113490786-51437700-9481-11eb-8a26-c4ff807c65e7.mp4

**Comment by davidcetinkaya** [comment]
Hi Clifford (@cliffordfajardo),

Please read my [response](https://github.com/davidcetinkaya/embla-carousel/issues/172#issuecomment-813008502) in issue #172, because this is related to an incorrect setup of Embla Carousel.

Best,
David

**Comment by davidcetinkaya** [comment]
Closing this as resolved based on your response in issue #172. Feel free to open this issue if you need further assistance.

Cheers,
David
